build-system/pr-check/build-targets.js

```diff
@@ -25,6 +25,28 @@ const config = require('../test-configs/config');
 const minimatch = require('minimatch');
 const path = require('path');
 const {gitDiffNameOnlyMaster} = require('../common/git');
+const {isTravisBuild} = require('../common/travis');
+
+/**
+ * List of all build targets.
+ */
+const ALL_TARGETS = [
```

Comment:
What if `targetMatchers` was refactored as a dictionary so we don't have to maintain this as well?